internal/test/framework/framework.go

```diff
@@ -40,35 +45,53 @@ type TestFramework struct {
 	SSHClient *ssh.Client
 	// CloudProvider holds the information related to cloud provider
 	cloudProvider cloudprovider.Cloud
+}
+
+// TestFramework holds the info to run the test suite.
+// This is not clean
```

Comment:
I don't know what `This is not clean` means in this context... since you're changing the struct can you fix the comment (if you can figure it out), or delete it?